app/Http/Controllers/Front/CartController.php

```diff
@@ -251,4 +259,12 @@ public function storeOrder(CartCheckoutRequest $order){
     public function thankYou(Order $order){
         return view('front.cart.thankyou')->with(['order'=> $order]);
     }
+
+    public function getWeights(){
+        $cartItems=$this->cartRepo->getCartItems()->pluck('options')->pluck('weight');
+        $multiplied = $cartItems->map(function ($item,$key){
+            return $item * $this->cartRepo->getCartItems()->pluck('qty')->get($key);
+        });
+        return $multiplied;
```

Comment:
Takto by to mozno tiez islo, treba si este dat pozor na pripady kedy je vaha null a to nejako osetrit.

```php
return $this->cartRepo->getCartItems()->map(function ($item){
            return $item['qty'] * $item['options']['weight'];
        });
```